packages/components/src/internal/components/search/QueryFilterPanel.tsx

```diff
@@ -59,12 +65,23 @@ export const QueryFilterPanel: FC<Props> = memo(props => {
         metricFeatureArea,
         fullWidth,
         selectDistinctOptions,
+        onHasNoValueInQueryChange,
+        hasNotInQueryFilterLabel,
     } = props;
     const [queryFields, setQueryFields] = useState<List<QueryColumn>>(undefined);
     const [activeField, setActiveField] = useState<QueryColumn>(undefined);
     const [activeTab, setActiveTab] = useState<FieldFilterTabs>(undefined);
 
     const queryName = useMemo(() => queryInfo?.name.toLowerCase(), [queryInfo]);
+
+    // for sample finder, assay data filters uses assay design name (part of schema key) instead of "data" (the query name) as query key
+    const filterQueryKey = useMemo(() => {
+        let queryKey = queryInfo?.name?.toLowerCase();
```

Comment:
Since it is calculated previously -- would need to add to dependency list too.
```suggestion
        let queryKey = queryName;
```